easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.5.1-intel-2016a-METIS-5.1.0.eb

```diff
@@ -0,0 +1,24 @@
+name = 'SuiteSparse'
+version = '4.5.1'
+
+homepage = 'http://www.cise.ufl.edu/research/sparse/SuiteSparse/'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+metis = 'METIS'
+metis_ver = '5.1.0'
+versionsuffix = '-%s-%s' % (metis, metis_ver)
+dependencies = [(metis, metis_ver)]
+
+preconfigopts = 'MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so" ' 
+prebuildopts = 'MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so" ' 
+preinstallopts = 'MY_METIS_LIB="$EBROOTMETIS/lib/libmetis.so" ' 
```

Comment:
@boegel Not 100% sure. The old `SuiteSparse` did not bundle `METIS 5.1.0`, this does. If you do not define the `MY_METIS_LIB` then it will be built and used. Maybe we have to solve this from the easyblock? (I was lazy)